Packs/Synapse/Integrations/Synapse/Synapse.yml

```diff
@@ -46,6 +46,10 @@ configuration:
   name: good_tag
   required: false
   type: 0
+- display: Use Synapse Optic
+  name: use_optic
+  required: false
+  type: 8
```

Comment:
Worth adding the 'additionalinfo' description for the 'use_optic' config parameter - similar to the description in all the other parameters.